libs/ledger-live-common/src/apps/polyfill.ts

```diff
@@ -97,13 +93,18 @@ export const getDependencies = (appName: string, appVersion?: string): string[]
 
 export const getDependents = (appName: string): string[] => reverseDep[appName] || [];
 
+function matchAppNameAndCryptoCurrency(appName: string, crypto: CryptoCurrency) {
+  return (
+    appName.toLowerCase() === crypto.managerAppName.toLowerCase() &&
+    (crypto.managerAppName !== "Ethereum" ||
+      // if it's ethereum, we have a specific case that we must only allow the Ethereum app
```

Comment:
🙈

Reply:
haha i just copied the existing logic here